.travis.yml

```diff
@@ -4,7 +4,7 @@
 # Some branches don't need builds.  Add them here to avoid using build
 # resources and unnecessary build messages.  See
 # <https://docs.travis-ci.com/user/conditions-v1>.
-if: NOT branch IN (branch/2023-01-07/pull-request-merge-procedure)
+if: NOT branch IN (branch/2023-01-13/rst-check)
 
```

Comment:
When this is done, there will be a merge conflict and we need to document how that should be addressed.

Reply:
The pull request merge procedure has provision for this situation.